Кузьмина Софья ТКИ-541 Курсовая работа

SonyaKuz opened the pull request.

SonyaKuz committed:
Add files via upload

SonyaKuz commented:
Сдаю черновик курсовой работы. С уважением, Кузьмина Софья, ТКИ-541.

MarinaPaley commented on orders.py:
```diff
@@ -0,0 +1,24 @@
+    quantity = Column(Integer, nullable=False)
+    total_price = Column(Float, nullable=False)
+
+    # ��������� �������� customer � sneaker ��� ����� � �������� Customer � Sneaker
```
Файл должен быть сохранен в кодировке `UTF-8`

MarinaPaley reviewed (changes requested):
Хорошо)

SonyaKuz committed:
Update orders.py

SonyaKuz committed:
Update reviews.py

SonyaKuz committed:
.

Здравствуйте, Марина Алексеевна. Сдаю Вам оставшиеся файлы. Все тесты работают, таблицы в Dbeaver появились. С уважением, Кузьмина Софья, ТКИ-541.

SonyaKuz committed:
Отчет по практической работе Кузьмина СофьяТКИ-541

Здравствуйте! Сдаю отчет по практической работе. С уважением, Кузьмина Софья, ТКИ-541.

MarinaPaley reviewed (changes requested):
1. Текст программы вставьте [моноширинным шрифтом](https://ru.wikipedia.org/wiki/%D0%9C%D0%BE%D0%BD%D0%BE%D1%88%D0%B8%D1%80%D0%B8%D0%BD%D0%BD%D1%8B%D0%B9_%D1%88%D1%80%D0%B8%D1%84%D1%82)
2. Источники надо вставлять [так](https://gist.github.com/MarinaPaley/1f83a9a15ae0a36f740eefe573a86f46), иначе, не пройдете антиплагиат

SonyaKuz committed:
Исправленный отчет

Исправленный отчет. Кузьмина СП, ТКИ-541

MarinaPaley reviewed (changes requested):
1. Надписи к рисункам "Вставка названия"
2. Список литературы также вставляем "не руками"

SonyaKuz committed:
исправляю список литературы
